spec/controllers/api/v3/facilities_controller_spec.rb

```diff
@@ -1,8 +1,8 @@
 require "rails_helper"
 
 RSpec.describe Api::V3::FacilitiesController, type: :controller do
+  before { Flipper.enable(:regions_prep) }
```

Comment:
same as above, or alternatively maybe a good approach is to have sub contexts in side the controller, one that enables the flag and one that disables?  Maybe it depends on the specifics of the context and setup required here.

Reply:
Addressed in https://github.com/simpledotorg/simple-server/pull/1333/commits/f343427699cf80a265bfc15ae0902876e78cd5d1?diff=unified&w=1